qa/L0_perf_analyzer/test.sh

```diff
@@ -63,7 +63,7 @@ for CONFIG_FILE in ${LIST_OF_CONFIG_FILES[@]}; do
         TEST_NAME="count_window"
     elif [[ "$CONFIG_FILE" == "config-additive-args-count-no-adjust.yml" ]]; then
         TEST_NAME="count_window"
-    elif [[ "$CONFIG_FILE" == "config_perf_output_timeout.yml" ]]; then
+    elif [[ "$CONFIG_FILE" == "config-perf-output-timeout.yml" ]]; then
```

Comment:
Is this an intended change?

Reply:
Yes. I have no idea how this ever passed before. You'll notice all other files have dashes instead of underscores. Without this TEST_NAME would be undefined and the check_results script will fail.

Reply:
I investigated how it was "passing" before. The TEST_NAME was unmodified from the previous test in the loop. I only discovered this because I locally edited the test to only execute a single test, so it ended up being undefined.